hyperstart_agent.go

```diff
@@ -411,6 +411,13 @@ func (h *hyper) startOneContainer(pod Pod, c *Container) error {
 		Process: process,
 	}
 
+	if c.config.Resources.VCPUs != 0 {
```

Comment:
You should test `c.config.Resources.CPUQuota` and `c.config.Resources.CPUPeriod` instead of `c.config.Resources.VCPUs`